components/brave_rewards/browser/test/rewards_p3a_browsertest.cc

```diff
@@ -132,6 +132,14 @@ class RewardsP3ABrowserTest : public InProcessBrowserTest,
     return;
   }
 
+  void TurnOnRewards(bool enable_ac = true) {
+    rewards_service_->SetAutoContributeEnabled(true);
```

Comment:
Should that be `rewards_service_->SetAutoContributeEnabled(enable_ac);`?

Reply:
I meant to remove that parameter - thanks for the catch!